util/openocd/target/lowrisc-earlgrey-silicon.cfg

```diff
@@ -0,0 +1,9 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+if { ![info exists CPUTAPID ] } {
```

Comment:
do you want the "if" block around this? Do you want to set the CPU TAP ID for good here?

Reply:
I think it's standard to allow an override via the if.